packages/components/inputs/rich-text-input/src/editor.tsx

```diff
@@ -107,8 +153,24 @@ const Editor = (props) => {
   );
 };
 
-// eslint-disable-next-line react/display-name
-const renderEditor = (props, editor, next) => {
+type TEditorOptionsProps = {
+  options: {
```

Comment:
Nit: it's a bit weird to type it like this. I would expect the `options` property to not be part of this type, then do

```ts
props: TEditorProps & { options: TEditorOptionsProps }
```

Reply:
true 🙂 updated